index.html

```diff
@@ -3,18 +3,41 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project | My Name (My GitHub username)</title>
+    <title>TV Show Project | Shadi Fakhri (Shadi38)</title>
     <link href="style.css" rel="stylesheet" />
     <link rel="icon" href="/favicon.ico" type="image/x-icon" />
   </head>
 
   <body>
-    <div id="root">
+     
+     
+    <div id="main-root">
+      <label>choose show:
+      <select id="select-show">
+        <option value="all show" selected>shows</option>
+      </select>
+      </label>
+      <lable>
+        choose episode:
+        <select id="select" class="margin-top-bottun">
+          <option value="all episodes" >all episodes</option>
+        </select>
+      </lable>
+      <input type="text" id="search-bar" name="search-bar"  placeholder="search" />
+      <label for="search-bar" id="lable"> Displaying 0/ 0 episodes</label>
+      <div id="div-btn">
+        <button id="btn">back to shows</button>
```

Comment:
I would make this id more specific so it is easy to understand what it does. Maybe call it "back-button"